Airbnb.Domain/Entities/Apartment.cs

```diff
@@ -7,6 +7,7 @@ public class Apartment
     public string Description { get; set; }
     public string Location { get; set; }
     public decimal Price { get; set; }
+    public string ExternalId { get; set; }
```

Comment:
You are missing indexes for external ids. There was an Acceptance Criteria to search by external ids, so better set up index for this field